docs/src/index.ejs

```diff
@@ -26,6 +26,7 @@
   </script>
 </head>
 <body>
+  <script src="https://cdn.jsdelivr.net/npm/@babel/polyfill@7/dist/polyfill.min.js"></script>
```

Comment:
Only for docs, see https://github.com/Semantic-Org/Semantic-UI-React/issues/3369